packages/open-next/src/build.ts

```diff
@@ -378,7 +378,7 @@ async function createImageOptimizationBundle(config: OpenNextConfig) {
   esbuildSync(
     {
       entryPoints: [path.join(outputPath, "index.mjs")],
-      external: ["sharp"],
+      external: ["sharp", "@opentelemetry/api"],
```

Comment:
Is `@opentelemetry/api` here as a baseline default for everyone? I haven't seen open-next's dependency on it, but I could have completely missed that.

Reply:
On the latest canary, this make the image optimization bundling crash. They probably added this as an optional deps inside Next and esbuild try to include it